docs/can-guides/commitment/recipes/playlist-editor/playlist-editor.md

```diff
@@ -488,18 +488,18 @@ In this section, we will:
   $(element).on('dropon', function(ev, drop, drag) {...});
   ```
 
-  Notice that `drag` is the 3nd argument to the event.  You can listen to
-  `drop` events in [can-stache] and pass the `drag` argument to a function like:
+  Notice that `drag` is the 2nd argument to the event.  You can listen to
```

Comment:
It should actually be pointing out the `drop` argument, I think. This is based on the fact that the `drag` event was already covered before this point. On line 386 of this markdown file, there is a very similar sentence:

>  Notice that `drag` is the 2nd argument to the event.  You can listen to
>  `drag` events in [can-stache] and pass the `drag` argument to a function like:

Unfortunately I can't link a line of a markdown file on GitHub.